### ISSUE TITLE
Future update

### ISSUE THREAD

**Issue by Bernard-h** [DustinBryant/MMM-Ring#6]
Hi Dustin, thanks for developing this module. My house is covered in ring cams, however I use my MagicMirror als communication hub for my household in the hallway. Thats right next to the door. I kinda like the idea, but presently id only install it if it can detects motion on cams around the property.

I have not seen or understand the unoffical ring api, but do you think you can implement the video capture for motion settings on normal ring cams (spotlight cam wired / wireless / floodlight cam). If you youd be an even bigger BOSS. 

Thanks!

**Comment by DustinBryant** [comment]
@Bernard-h thanks for your interested in this module. I had created this module in a private repo, but then saw interest in it on the MagicMirror forums as I was creating my own MagicMirror. I figured I'd make it public for those who may find it helpful. 

My use case was for door bell ring events only and did not need or want my other camera's motion video. Unfortunately, I still see no need for that feature for myself. As such, I don't think it would be too difficult to alter the code and add motion video responses. The challenge anyone who takes this on would be to prioritize which video to show at any given time since multiple motion events could happen at the same time. Again, not a road I'm looking to venture down based solely on my needs for this module. 

Alas, I'm also looking to get rid of my Ring devices here shortly in favor of an open source and locally managed cameras. As such, this module will eventually become obsolete unless someone takes over this repository for further development since I won't have the Ring devices to debug issues.

Because of the aforementioned departure in the near future, I'll mark this issue as an enhancement just in case whoever takes this repo over may have interest in this feature.
